app/src/main/java/com/mapbox/maps/testapp/examples/LegacyOfflineActivity.kt

```diff
@@ -19,7 +19,7 @@ class LegacyOfflineActivity : AppCompatActivity() {
   private var mapView: MapView? = null
   private lateinit var binding: ActivityLegacyOfflineBinding
 
-  private val regionObserver: OfflineRegionObserver = object : OfflineRegionObserver() {
+  private val regionObserver: OfflineRegionObserver = object : OfflineRegionObserver {
```

Comment:
nit:
```suggestion
  private val offlineRegionObserver: OfflineRegionObserver = object : OfflineRegionObserver {
```

Reply:
I'll not be doing refactoring existent code as part of this PR, this PR is to adopt to latest core only.
We can revisit those nit's later as part of some other cleanup.